spec/acceptance/nodesets/default.yml

```diff
@@ -11,7 +11,7 @@ HOSTS:
       - server
       - default
     platform:   el-8-x86_64
-    box:        centos/8
```

Comment:
This should remain at `centos/8` unless there's a functional reason why it won't work.

If there is, please add a comment as to why it has been changed.

Reply:
Nope, this was my oversight; guest plugin doesn't work locally without additional settings (`vbguest.installer_options`) for `centos/8` image due to repo names, etc. so I used `generic/centos8` locally for testing and pushed it up accidentally.